### PR TITLE
Connect to api

### PR DESCRIPTION
**Pull request by Mesi21**
- Connect to API

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -32,9 +29,55 @@ const booksReducer = (state = stateInit, action) => {
...
+      title: newBook.title,
+      author: newBook.author,
+      category: newBook.category,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  dispatch(addBook(newBook));
+};
```
Comment (nicupop729) [inline src/redux/books/books.js:68]: If I add a new book, I can clearly see that is straight away added to the list. But unfortunately, it is not persisting; on refresh, it disappears.

- [x] Kindly check what exactly the issue is. First, you can check if it is added to the API. If it is, let's see why you are not getting it after refresh.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -32,9 +29,55 @@ const booksReducer = (state = stateInit, action) => {
...
+    body: JSON.stringify({
+      item_id: id,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+  dispatch(removeBook(id));
+};
+
```
Comment (nicupop729) [inline src/redux/books/books.js:82]: The almost same thing happens here: if I delete a book, it disappears from the list. But after refreshing, I see it back again. Unfortunately, the API is not informed about the changes.

- [x] Kindly check what the issue may be regarding this aspect and when you are ready, please submit again. :+1:

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @Mesi21, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - [X] Correct git flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Professional README :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @Mesi21, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**